test/e2e/specs/onboarding/ftme__site-assembler.ts

```diff
@@ -118,15 +118,21 @@ describe( 'Site Assembler', () => {
 			await siteAssemblerFlow.clickButton( 'Save and continue' );
 		} );
 
-		it( 'Click "Save and continue" and land on the Site Editor', async function () {
-			const urlPromise = page.waitForURL( /processing/ );
+		it( 'Click "Start adding content" to the next step', async function () {
 			await siteAssemblerFlow.clickButton( 'Start adding content' );
-			await urlPromise;
+		} );
 
-			await page.waitForURL( /site-editor/, {
+		it( 'Land on the Celebration step', async function () {
+			await page.waitForURL( /processing/ );
+			await page.waitForURL( /celebration-step/, {
 				timeout: 30 * 1000,
 			} );
 		} );
+
+		it( 'Click "Edit your content" and land on the Site Editor', async function () {
+			await siteAssemblerFlow.clickButton( 'Edit your content' );
+			await page.waitForURL( /site-editor/ );
```

Comment:
**suggestion:blocking** - I'm going to strongly suggest we add or move the 30s timeout to the `waitForURL( /site-editor/ )` portion, because the loading can take much longer than the 10s default timeout.

Reply:
Okay 👍